Controllers/FHIRcastClientController.cs

```diff
@@ -179,14 +194,14 @@ public IActionResult Post(string subscriptionId, [FromBody] Notification notific
             return View("FHIRcastClient", internalModel);
         }
 
-        [Route("unsubscribe/{subscriptionId}")]
+        [Route("Unsubscribe/{subscriptionId}")]

```

Comment:
Is this suggestion correct? Routes are generally lower-case and I think we should stick to that convention (even if it doesn't matter, technically).